<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerFeetSegmentationIOWidget</class>
 <widget class="QWidget" name="qSlicerFeetSegmentationIOWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="inputBox">
     <property name="title">
      <string>Input</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="rgbInputLabel">
          <property name="text">
           <string>RGB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="qMRMLNodeComboBox" name="rgbInputNodeSelector" native="true"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="depthInputLabel">
          <property name="text">
           <string>Depth</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="qMRMLNodeComboBox" name="depthInputNodeSelector" native="true"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="outputBox">
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="resultOutputLabel">
          <property name="text">
           <string>Result</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="qMRMLNodeComboBox" name="resultOutputNodeSelector" native="true"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
